Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -469,3 +469,31 @@
   components:
   - type: PDA
     id: CentcomIDCardSyndie
+
+- type: entity
+  parent: BasePDA
+  id: MusicianPDA
+  name: musician PDA
+  description: It fills you with inspiration.
+  components:
+  - type: PDA
+    id: MusicianIDCard
+  - type: Appearance
+    visuals:
+    - type: PDAVisualizer
+      state: pda-musician
+  - type: Instrument
+    allowPercussion: false
+    handheld: true
+    bank: 1
+    program: 2
+  - type: ActivatableUI
+    inHandsOnly: true
+    singleUser: true
+    key: enum.InstrumentUiKey.Key
+  - type: UserInterface
+    interfaces:
+    - key: enum.InstrumentUiKey.Key
+      type: InstrumentBoundUserInterface
```

Comment:
Looks like this got rid of the BUIs by mistake.
Add
```
    - key: enum.PDAUiKey.Key
      type: PDABoundUserInterface
    - key: enum.UplinkUiKey.Key
      type: UplinkBoundUserInterface
    - key: enum.RingerUiKey.Key
      type: RingerBoundUserInterface
```
after this.

Reply:
#6932 for your consideration